test/reduce-e2e/testdata/joined-map-vertex-pipeline.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: joined-map-vertex
```

Comment:
Should this be named to 'join-on-map' to match the naming pattern of the above reduce join vertex pipeline? So is the file name.